unified-codes/apps/web/src/components/molecules/EntityTypeFilter.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { ToggleButtonGroup } from '@unified-codes/ui';
+
+export type EntityTypeFilter = React.FunctionComponent;
+
+export const EntityTypeFilter: EntityTypeFilter = () => {
+  const [entityStates, entityStateChange] = React.useState([
+    { name: 'Drugs', active: true },
+    { name: 'Unit of Use', active: false },
```

Comment:
`Drugs` is plural whereas `Unit of Use` and `Other` are singular - we need to decide what to have here I think
Also - `Drug` is top level in the hierarchy and `Unit of Use` is much lower down - given that we only show top level items, probably doesn't make sense to have both together in a filter.

Not a coding comment, but something that's been troubling me. any ideas how we manage this?

Reply:
Hmm, I've also been a little unsure about the use case. I was also thinking it doesn't make sense to have both, but was thinking maybe people would be more interested in seeing unit of uses in the table rather than the medicinal product level? But if someone clicked through on a specific product they'd then be able to browse the hierarchy? Or something to that effect...? 🤷‍♀️